src/components/common/TmBalance.vue

```diff
@@ -248,6 +250,30 @@ export default {
       },
       /* istanbul ignore next */
       update(data) {
+        if (!this.sentToGA) {
+          // sending to ga only once
+          sendEvent(
+            {
+              network: this.network,
+              address: this.address
+            },
+            "Portfolio",
+            "Balance",
+            "liquidStake",
+            data.overview.liquidStake
```

Comment:
should we just include the whole overview object?

would be nice to have rewards balance as well - so we can know unclaimed rewards.

Reply:
We can't include object itself, cause it heeds to be a number, but we can add rewards as well

Reply:
yeah, please add rewards.